.github/workflows/scala-deploy-web.yml

```diff
@@ -33,7 +33,7 @@ on:
 
 jobs:
   testing:
-    if: inputs.environment == 'testing'
+    if: github.event.inputs.environment == 'testing'
```

Comment:
@jonathan-correa-paiva verifiquemos esto!! por las dudas!!!

Reply:
github.event.inputs se utiliza cuando es por [workflow_dispatch](https://docs.github.com/es/enterprise-cloud@latest/actions/using-workflows/workflow-syntax-for-github-actions#onworkflow_dispatchinputs), para [workflow_call](https://docs.github.com/es/enterprise-cloud@latest/actions/using-workflows/workflow-syntax-for-github-actions#onworkflow_callinputs) se utiliza inputs solo.

Lo dejo como inputs, ya que no se debería utilizar dispatch, eso fue con la prueba que hice en el otro repo.